packages/generator-langium/templates/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "<%= extension-name %>-base",
+    "description": "Base workspace package",
```

Comment:
Maybe change the `base` into `monorepo` or `workspace`?

Reply:
Good idea. Will use workspace.

Reply:
Done ⬇️